app/src/main/java/com/igordotsenko/dotsenkorssreader/MainActivity.java

```diff
@@ -10,68 +17,94 @@
 import android.view.View;
 import android.widget.ImageButton;
 
+import static com.igordotsenko.dotsenkorssreader.ReaderContentProvider.ContractClass;
 import com.igordotsenko.dotsenkorssreader.adapters.ChannelListRVAdapter;
-import com.igordotsenko.dotsenkorssreader.entities.Channel;
-import com.igordotsenko.dotsenkorssreader.entities.DBHandler;
 import com.nostra13.universalimageloader.core.ImageLoader;
 import com.nostra13.universalimageloader.core.ImageLoaderConfiguration;
 
-import java.util.ArrayList;
-import java.util.List;
+public class MainActivity extends AppCompatActivity
+        implements SearchView.OnQueryTextListener, LoaderManager.LoaderCallbacks<Cursor> {
 
+    private static final String QUERY_TEXT = "query text";
+    private static final int LOADER_CHANNEL_LIST = 1;
+    private static final int LOADER_CHANNEL_LIST_REFRESH = 2;
+    public static final String LOG_TAG = "rss_reader_log";
+    public static final String DB_NAME = "rss_reader.db";
+    public static final int DB_VERSION = 1;
+    public static final String ACCOUNT_TYPE = "com.igordotsenko.dotsenkorssreader";
+    public static final String ACCOUNT = "dummyaccount";
 
-public class MainActivity extends AppCompatActivity implements SearchView.OnQueryTextListener {
-    private DialogFragment dialogFragment;
-    private RecyclerView recyclerView;
-    private SearchView searchView;
-    private ImageButton addChannelButton;
-    private List<Channel> channelList;
-    private ChannelListRVAdapter rvAdapter;
+    public static Account sAccount;
+
+    private DialogFragment mDialogFragment;
+    private RecyclerView mRecyclerView;
+    private SearchView mSearchView;
+    private ImageButton mAddChannelButton;
+    private ChannelListRVAdapter mRvAdapter;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
         setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
 
+        // Creating sAccount for SyncAdapter
+        sAccount = new Account(ACCOUNT, ACCOUNT_TYPE);
```

Comment:
Strange logic, you have static variable but creating new account each time instantiating main activity.